lang/LangPrimSource/PyrPrimitive.cpp

```diff
@@ -4264,7 +4264,7 @@ void initMIDIPrimitives();
 	initMIDIPrimitives();
 #endif
 
-#if !defined(_WIN32) && !defined(SC_IPHONE) && !defined(__OpenBSD__) && !defined(__NetBSD__) && !defined(__APPLE__)
+#if defined __linux__
```

Comment:
I raised the question because I wanted this to be a conscious decision, rather than because I had strong views. I'll go along with your argument, yes. No need to hold up the merge.